azext_iot/sdk/iothub/service/iot_hub_gateway_service_ap_is.py

```diff
@@ -85,21 +93,25 @@ def __init__(
         super(IotHubGatewayServiceAPIs, self).__init__(self.config.credentials, self.config)
 
         client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
-        self.api_version = '2019-10-01'
+        self.api_version = '2020-06-30-preview'
```

Comment:
There is a mix of API versions? `2020-09-30` and `2020-06-30-preview`? Can we just stick with `2020-09-30`?

Reply:
my bad. this folder went through api update twice, but the second time i left out the parent folder